modules/publicize/class-async-publicize.php

```diff
@@ -0,0 +1,343 @@
+<?php
+/**
+ * Provides REST endpoints to accommodate asynchronous Publicize use
+ *
+ * Provides REST endpoints to publish a post without publicizing and to publicize
+ * an already published post. Publicize was originally written to publicize
+ * a post right as it's being published. This class extends the standard
+ * Publicize behavior to accommodate user interface designs
+ * that separate publish and publicize actions (i.e. in Gutenberg).
+ *
+ * @package Jetpack
+ * @subpackage Publicize
+ * @since 5.9.1
+ */
+
+/**
+ * Class to set up asynchronous (publish and later publicize) support.
+ *
+ * This class sets up REST endpoints for publishing posts and then
+ * later sharing them with Publicize.
+ *
+ * @since 5.9.1
+ */
+class Async_Publicize {
```

Comment:
Lets prefix this with Jetpack.

Reply:
So, just to be clear, "Jetpack_Async_Publicize"?